### PR TITLE
Support absolute imports + style lint script fix

### PR DESCRIPTION
**Pull request by evgenyboxer**
In newer versions of `create-react-app` its a lot easier... but since we ejected we need to have some manual workaround.

```
import { DataMedium } from '../../../../../../components/Typography';

becomes:

import { DataMedium } from 'src/components/Typography';
```

### CODE REVIEW

File: jsconfig.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"compilerOptions": {
+		"baseUrl": ".",
+		"paths": {
+			"src": ["src/*"]
+		}
+	}
+}
```
Comment (evgenyboxer) [inline jsconfig.json]: this file allows code editors to autocomplete from `src/...`

File: package.json [hunk 2]
```diff
@@ -40,8 +40,8 @@
 		"start": "node --max-old-space-size=4096 scripts/start.js",
 		"build": "node scripts/build.js",
 		"test": "node scripts/test.js",
-		"lint": "eslint .",
-		"lint:css": "stylelint src/**/**/*.js",
+		"lint": "eslint 'src/**/*.js'",
+		"lint:css": "stylelint 'src/**/*.js'",
```
Comment (evgenyboxer) [inline package.json]: fixes style linting

File: src/shared/commonStyles.js [hunk 3]
```diff
@@ -144,7 +144,6 @@ export const InfoBox = styled.div`
 
 export const InfoBoxLabel = styled(DataSmall)`
 	white-space: nowrap;
-	text-transform: none;
```
Comment (evgenyboxer) [inline src/shared/commonStyles.js]: already caught one!

File: .eslintrc [hunk 4]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (evgenyboxer) [inline .eslintrc]: switched to a JSON file

**Comment by clementbalestrat** [comment]
Great idea. I'm going to do the same in Mintr.

**Pull request merged**